src/components/search/Autocomplete.tsx

```diff
@@ -10,7 +10,11 @@ import '@algolia/autocomplete-theme-classic'
  */
 export const Autocomplete = (props): JSX.Element => {
   const containerRef = useRef<HTMLDivElement>(null)
-
+  useEffect(() => {
+    setTimeout(() => {
+      (document.getElementsByClassName('aa-Input')[0] as HTMLInputElement).focus()
+    }, 100)
+  }, [])
```

Comment:
I believe the culprit was 
className="ease-in-out duration-500"

from [develop/src/components/search/Tabs.tsx](https://github.com/clintonlunn/open-tacos/blob/develop/src/components/search/Tabs.tsx)